Gruntfile.js

```diff
@@ -18,7 +18,7 @@ module.exports = function (grunt) {
 					mainFile  : 'backupwordpress.php',         // Main project file.
 					type      : 'wp-plugin',    // Type of project (wp-plugin or wp-theme).
 					processPot: function( pot, options ) {
-						pot.headers['report-msgid-bugs-to'] = 'support@humanmade.co.uk';
+						pot.headers['report-msgid-bugs-to'] = 'backupwordpress@hmn.md';
```

Comment:
Wat, where does this come from. Also the line is shorter than the one it replaces...